<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DataSetFilterView</class>
 <widget class="QWidget" name="DataSetFilterView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1351</width>
    <height>995</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab_inputGroups">
      <attribute name="title">
       <string>1 Select Recording Groups</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout" columnstretch="0,1,3">
       <item row="0" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Search:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="lineEdit_inputSearch"/>
       </item>
       <item row="0" column="2" rowspan="2">
        <widget class="QListWidget" name="listWidget_inputRecordings"/>
       </item>
       <item row="1" column="0" colspan="2">
        <widget class="QListWidget" name="listWidget_inputGroups"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_filters">
      <attribute name="title">
       <string>2 Filter Recordings</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_2" columnstretch="4,2">
       <item row="1" column="0">
        <widget class="QScrollArea" name="scrollArea_filters">
         <property name="widgetResizable">
          <bool>true</bool>
         </property>
         <widget class="QWidget" name="scrollAreaWidgetContents_filters">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>871</width>
            <height>874</height>
           </rect>
          </property>
         </widget>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QListWidget" name="listWidget_outputGroup"/>
       </item>
       <item row="0" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QToolButton" name="toolButton_addFilter">
           <property name="text">
            <string>Add Filter</string>
           </property>
           <property name="popupMode">
            <enum>QToolButton::InstantPopup</enum>
           </property>
           <property name="toolButtonStyle">
            <enum>Qt::ToolButtonTextBesideIcon</enum>
           </property>
           <property name="arrowType">
            <enum>Qt::DownArrow</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_savePreset">
           <property name="text">
            <string>Save as Preset</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_loadPreset">
           <property name="text">
            <string>Load Preset</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item row="0" column="1">
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Data Set:</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_createAnalysis">
           <property name="text">
            <string>Create New Analysis</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
  <action name="actionStage">
   <property name="text">
    <string>Stage</string>
   </property>
  </action>
  <action name="actionMatchup">
   <property name="text">
    <string>Matchup</string>
   </property>
  </action>
  <action name="actionPlayer">
   <property name="text">
    <string>Player</string>
   </property>
  </action>
  <action name="actionDate">
   <property name="text">
    <string>Date</string>
   </property>
  </action>
  <action name="actionGame">
   <property name="text">
    <string>Game</string>
   </property>
  </action>
  <action name="actionPlayerCount">
   <property name="text">
    <string>PlayerCount</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
